src/XCCDF/public/xccdf_session.h

```diff
@@ -42,6 +42,19 @@
 struct xccdf_session;
 
 /**
+ * Loading flags for XCCDF session
+ * @memberof xccdf_session
+ * The flags set which components will be loaded in XCCDF session.
+ */
+typedef enum {
+	XCCDF_SESSION_LOAD_NONE = 0,
```

Comment:
The problem is that since it's zero there is no way to "remove" it from the bitfield. Maybe it would be enough to have _NONE there but have it documented that it does load the XCCDF and will load parse the DS to get the XCCDF out of it.